docs/artifactory/config/help.go

```diff
@@ -13,6 +15,12 @@ const Arguments string = `	server ID
 	show
 		Shows the stored configuration.
 		In case this argument is followed by a configured server ID, then only this server's configurations is shown.
+	
+	import
+		Imports an artifactory server configuration from server token.
```

Comment:
Imports an artifactory server configuration from server token. -->
Imports an Artifactory server configuration from a server token. A server token is generated by the "jfrog rt c export [Server ID]" command.